src/goto-programs/remove_const_function_pointers.cpp

```diff
@@ -784,7 +784,9 @@ bool remove_const_function_pointerst::is_const_expression(
 ///   arrays are implicitly const in C.
 bool remove_const_function_pointerst::is_const_type(const typet &type) const
 {
-  if(type.id() == ID_array && type.subtype().get_bool(ID_C_constant))
+  if(
+    type.id() == ID_array &&
+    to_array_type(type).element_type().get_bool(ID_C_constant))
     return true;
 
   return type.get_bool(ID_C_constant);
```

Comment:
Nit pick: would appreciate a pair of { ... } given the multi-line condition. Or perhaps just rewrite this function to just have a `return` statement.

Reply:
done